users/migrations/0012_allow_duplicate_account_emailaddresses.py

```diff
@@ -0,0 +1,21 @@
+from django.db import migrations
```

Comment:
This shouldn't be included in this MR.

Reply:
I already did this change yesterday but forgot to push changes here. Anyway, migration file is now removed